.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: ci
+on: push
```

Comment:
Do we want the process to start per push? Maybe we want it to trigger only while pushing to the main branch.

Reply:
Changed to PR